pkg/cloudclient/gcp/private.go

```diff
@@ -212,6 +215,7 @@ func (c *Client) findGCPForwardingRuleForExtIP(rhapiLbIP string) (*compute.Forwa
 	for _, lb := range response.Items {
 		if lb.IPAddress == rhapiLbIP {
 			fr = lb
+			break
 		}
 	}
 	return fr, nil
```

Comment:
```suggestion
	return fr, fmt.Errorf("no forwarding rule found for rhapi IP: %v", rhapiLbIP)
```

Reply:
Also, should https://github.com/openshift/cloud-ingress-operator/blob/31bb6af09a1e6ab4e8fee6350da195ac16b8085a/pkg/errors/errors.go#L25-L29 be cleaned up a bit to handle either the GCP api error, or this one?

Reply:
This is only going to happen in GCP. We can change "not found in cloud provider" to "not found in GCP".

Reply:
Updated